etc/config.yaml.template

```diff
@@ -0,0 +1,4 @@
+# Peregrine Configuration
+server:
+  listen: :8080
+  origin: \*
```

Comment:
Can you rename `listen` to `port`? Also could it be `8080` instead of `:8080`?

What is `origin`?

Can you add a section about copying this file, and a description of the fields in the README?

Reply:
It's not port, it's the address to list on. Origin is the origin for Access-Control-Allow-Methods, and yes I'll document that in the YAML file with comments in the template.